tests/integration/s2n_handshake_test_gnutls.py

```diff
@@ -36,8 +36,8 @@ def try_gnutls_handshake(endpoint, port, priority_str):
     s2nd.stdout.readline()
 
     # Fire up gnutls-cli, use insecure since s2nd is using a dummy cert
-    gnutls_cli = subprocess.Popen(["gnutls-cli", "--priority=" + priority_str,"--insecure", "-p " + str(port), str(endpoint)], 
-                                   stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.DEVNULL)
+    gnutls_cli = subprocess.Popen(["gnutls-cli", "--priority=" + priority_str,"--insecure", "--recordsize=" + "512",
```

Comment:
Thanks for adding a working integration test with GnuTLS client! A couple suggestions:
- Does gnutls-cli succeed if the server ignores the MFL extension? If it does, this test may succeed even though our server did not actually negotiate MFL.
- This change applies to all of the GnuTLS tests since we've hardcoded it into `try_gnutls_handshake`. We lose the test coverage with GnuTLS and no MFL. Can you update this change to continue testing non-mfl? You could make `mfl` an optional argument to `try_gnutls_handshake` and then add a function to test mfl specifically. 
- Let's iterate over each mfl in this test: {512, 1024, 2048, 4192}.